childController/selectionOfFormula.py

```diff
@@ -0,0 +1,16 @@
+class Selection:
+    def selectOption(self):
+        while True:
+            try:
+                option= int(input("Enter the option you want: "))
```

Comment:
Take **option** variable first before the loop or better as a **class** variable first.